config/src/main/java/org/springframework/security/config/annotation/web/builders/HttpSecurity.java

```diff
@@ -1342,6 +1342,21 @@ public HttpSecurity authorizeHttpRequests(
 		return HttpSecurity.this;
 	}
 
+	/**
+	 * get authorizeHttpRequests with default values
+	 * @return the {@link HttpSecurity} for further customizations
+	 * @throws Exception
+	 * @see #authorizeHttpRequests(Customizer)
+	 */
+	public HttpSecurity authorizeHttpRequests() throws Exception {
```

Comment:
I think it would be good to add at least one test into `AuthorizeHttpRequestsConfigurerTests` to confirm that the method works.

Reply:
Added it's under deployed

Reply:
Tests added